RELEASE-NOTES.txt

```diff
@@ -4,6 +4,7 @@
 * [*] Stats: fix navigation between Stats tab. [#17894]
 * [*] Add "Copy Link" functionality to Posts List and Pages List [#17911]
 * [*] [Jetpack-only] Enables the ability to use and create WordPress.com sites, and enables the Reader tab. [#17914, #17948]
+* [*] Block editor: Autocorrected Headings no longer apply bold formatting if they weren't already bold. [#17844]
```

Comment:
If the Gutenberg Mobile ref is not being updated, which would include this fix, I'm wondering whether we should add this entry, wdyt?

Reply:
That's a good point @fluiddot - I'd leave it out if we were just updating `WordPress-Editor-iOS`. After our discussion, though, the plan will be to include the GB Mobile upgrade as well with this PR.